shell/platform/android/android_surface_gl.cc

```diff
@@ -125,4 +125,9 @@ intptr_t AndroidSurfaceGL::GLContextFBO() const {
   return 0;
 }
 
+// |GPUSurfaceGLDelegate|
+ExternalViewEmbedder* AndroidSurfaceGL::GetExternalViewEmbedder() {
```

Comment:
Since not all platforms implement platform views like iOS and embedders do, it was not immediately clear from looking at just one implementation if the technique was used on that platform (Android in this case). I believe this explicitness makes it more clear from looking at the Android implementation. Also, this happened to be the only function in the delegate that had a default implementation in the base class (the acquire and present calls on the backing store were pure virtual).

Reply:
sgtm